atef/widgets/config.py

```diff
@@ -1428,6 +1431,32 @@ def edit_items(self, old_items: List[str], new_items: List[str]) -> None:
             else:
                 self._edit_item(old, new)
 
+    def _show_context_menu(self, pos):
+        """
+        Displays a context menu that provides copy & remove actions
+        to the user
+
+        Parameters
+        ----------
+        pos : QPoint
+            Position to display the menu at
+        """
+        menu = QMenu(self)
+
+        def copy_selected():
+            items = self.list_strings.selectedItems()
+            if len(items) > 0:
+                QGuiApplication.clipboard().setText(items[0].text(),
```

Comment:
Multi-selection is possible, but we only copy the first item here?

Reply:
I wasn't sure what the user would expect. Right click -> Copy on multiple items only copies the right clicked item, or all of them? Either works for me.

Reply:
Personally I'd expect it would copy all of the ones I've selected, perhaps with a newline delimiter. Anyone else feel differently?